### PR TITLE
fix menu placement is broken on mobile

### LINKED ISSUE #3631: iPad - Login - App freezes after login in landscape orientation
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app in landscape orientation
2. Login with valid credentials

## Expected Result:
User should be taken to the home page

## Actual Result:
The chat is displayed for a second and then the app freezes

## Workaround:
Unknown

## Platform:
Where is this issue occurring?

 Web
 iOS ✔️
 Android
 Desktop App
 Mobile Web

**Version Number:** 1.0.70-0
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/122312711-cea85080-cee2-11eb-90e7-18c7bb81ec5e.mp4

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### LINKED ISSUE #3633: Android/iOS - Chat - Long pressing on sent message doesn't open menu properly
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app
2. Log in with any account
3. Go to any conversation
4. Find your messages and long press

## Expected Result:
The flyout menu should be displayed properly at the bottom of the screen and all 4 options (Copy to clipboard, Mark as unread, Edit comment and Delete comment) should be displayed

## Actual Result:
On iOS, the menu is displayed in the middle of the screen and only 2 options are displayed. On Android, there are only 2 options displayed (Edit comment and Mark as unread) and overlap with the message.

## Workaround:
Unknown

## Platform:
Where is this issue occurring?

 Web
 iOS✔️
 Android✔️
 Desktop App
 Mobile Web

**Version Number:** 1.0.70-0
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/122315398-fb129b80-cee7-11eb-9e5b-435f74f2f9b0.mp4

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### LINKED ISSUE #3636: mWeb - User is able to open several action menus for messages
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Log in to e.cash in mobile web
2. Navigate to a chat
3. Press and hold to trigger the action menu in several messages. 

## Expected Result:
Action menu should be dismissed when opening the menu in another message. 

## Actual Result:
Several action menus can be opened. 

## Workaround:
Unknown 

## Platform:
Where is this issue occurring?

 Web
 iOS
 Android
 Desktop App
 Mobile Web ✔️ 

**Version Number:** 1.0.70-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:**

https://user-images.githubusercontent.com/44479856/122421719-7cb80700-cf84-11eb-85a0-21c6ab35a9a3.mp4

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by parasharrajat**
### Details
Reverted accidentally deleted file. and refactor Code for IOS.

### Fixed Issues

Fixes #3633
Fixes #3636
fixes #3631

### Tests | QA Steps
1. Open any chat on Mobile.
2. open context menu on any message.
3. It should open correctly from bottom to top.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/24370807/122476215-70af6200-cfe3-11eb-9a38-66476601f403.mp4

#### Mobile Web

https://user-images.githubusercontent.com/24370807/122443202-f79d1400-cfbc-11eb-922c-cee46082cc9b.mp4

#### iOS

https://user-images.githubusercontent.com/24370807/122443467-3df27300-cfbd-11eb-81fc-731eb991617c.mp4

### CODE REVIEW

**Comment by parasharrajat** [comment]
It's a little urgent and I believe needs to go to Production.

**Comment by roryabraham** [comment]
@parasharrajat Please provide screenshots for this PR

**Comment by parasharrajat** [comment]
Uploading

**Comment by parasharrajat** [comment]
@roryabraham Updated.

File: src/components/Popover/index.native.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        // eslint-disable-next-line react/jsx-props-no-multi-spaces
+        fullscreen
+    />
+);
+
+Popover.propTypes = propTypes;
+Popover.defaultProps = defaultProps;
+Popover.displayName = 'Popover';
+
+export default withWindowDimensions(Popover);
```
Comment (roryabraham) [inline src/components/Popover/index.native.js:27]: Can we safely get rid of `withWindowDimensions` here? Seems unnecessary.

**Review by roryabraham — CHANGES_REQUESTED** [review]

**Review by Luke9389 — DISMISSED** [review]
I was the one who deleted the `.native` index file, but now I see that we need both because of the `ReactDOM.createPortal` usage. I think it was just bad timing that we did this at the same time.

**Comment by parasharrajat** [comment]
Updated. Let's merge this quickly

**Comment by parasharrajat** [comment]
Let's merge this @roryabraham It's fixing deploy blockers.

**Review by roryabraham — APPROVED** [review]

**Comment by roryabraham** [comment]
Heads up I think this may have caused a regression where the global create menu is sliding in right-to-left

**Comment by parasharrajat** [comment]
Oh. Sending another PR. Didn't notice that part.

**Comment by parasharrajat** [comment]
Follow-up PR is ready here #3684.  Thanks

**Pull request merged**